projects/pastanaga/src/lib/translate/translate.directive.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { TranslateDirective } from './translate.directive';
+import { TranslatePipe } from './translate.pipe';
+import { ElementRef } from '@angular/core'; 
+class MockElementRef implements ElementRef {
+    nativeElement = {};
+}
+const obj1 = {common: 1, second: 2};
+const obj2 = {common: 1, color: 'blue' };
+const obj3 = {common: 1, second: 3};
+
+describe('TranslateDirective', () => {
+  it('should create an instance', () => {
+    const directive = new TranslateDirective(new MockElementRef(), new TranslatePipe({}));
+    expect(directive).toBeTruthy();
+  });
+});
+
+describe('TranslateDirective', () => {
+    it('should be false', () => {
```

Comment:
you need to describe what you are testing here, so if the test fails we can have an idea of what is wrong just by reading the unit tests output.